docs/Project.toml

```diff
@@ -1,6 +1,7 @@
 [deps]
+DataStructures = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+SiennaTemplate = "bed98974-b02a-5e2f-9ee0-a103f5c450dd"
 
```

Comment:
One more add:
[DocumenterInterLinks.jl](http://juliadocs.org/DocumenterInterLinks.jl/stable/#DocumenterInterLinks.jl)

We aren't currently using this, but I would like us to

Reply:
Added